agent/command/archive_util.go

```diff
@@ -222,14 +221,137 @@ func TarGzReader(path string) (f, gz io.ReadCloser, tarReader *tar.Reader, err e
 	return f, gz, tarReader, nil
 }
 
-// TarGzWriter returns a file, gzip writer, and tarWriter for the path.
+// tarGzWriter returns a file, gzip writer, and tarWriter for the path.
```

Comment:
Could this be updated to explain useParallelGzip?

Reply:
Done.